boards/bluepill/include/board.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2015 TriaGnoSys GmbH
+ *               2017 Alexander Kurth, Sören Tempel, Tristan Bruns
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_bluepill Bluepill board
+ * @ingroup     boards
+ * @brief       Support for the stm32f103c8 based bluepill board.
+ * @details     This board can be bought very cheaply on sides like eBay
```

Comment:
add empty line and omit `@details` - IMHO its not needed when separated from `@brief` by an empty line